src/main/java/seedu/address/model/person/Priority.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Food item priority level in the food inventory system.
+ * Guarantees: immutable; is valid as declared in {@link #isValidPriority(String)}
+ */
+public class Priority {
+    public enum Level {
+        LOW,
+        MEDIUM,
+        HIGH
+    }
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Priority should be either high, medium or low.";
```

Comment:
Just to standardize with the other fields: "Priorities should be either high, medium or low.";

Reply:
Alright will change.